acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/permissioning/PermissionedNodeBuilder.java

```diff
@@ -209,12 +210,15 @@ private LocalPermissioningConfiguration localConfigPermissioningConfiguration()
         localConfigNodesPermissioningFile = createTemporaryPermissionsFile();
       }
 
-      List<String> nodesAsListOfStrings =
+      final List<String> nodesAsListOfStrings =
```

Comment:
nit; these local variables could be have more succinct  names (as we know they are list of strings from typing).

Maybe something like `localPermittedNodes` and `localPermittedEnodeUrls`?

Reply:
it's changed 👍